aws/table_aws_ssm_association.go

```diff
@@ -0,0 +1,204 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ssm"
+
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+)
+
+func tableAwsSSMAssociation(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_ssm_association",
+		Description: "AWS SSM Association",
+		Get: &plugin.GetConfig{
+			KeyColumns:        plugin.SingleColumn("association_id"),
+			ShouldIgnoreError: isNotFoundError([]string{"AssociationDoesNotExist"}),
+			Hydrate:           getAwsSSMAssociation,
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listAwsSSMAssociations,
+		},
+		GetMatrixItem: BuildRegionList,
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "association_id",
+				Description: "The ID created by the system when you create an association.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "association_name",
+				Description: "The Name of association.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "document_name",
+				Description: "The name of the Systems Manager document.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Name"),
+			},
+			{
+				Name:        "date",
+				Description: "The date when the association was made.",
+				Hydrate:     getAwsSSMAssociation,
+				Type:        proto.ColumnType_DATETIME,
```

Comment:
Use `proto.ColumnType_TIMESTAMP` instead of using `proto.ColumnType_DATETIME`
```suggestion
				Type:        proto.ColumnType_TIMESTAMP,
```